_includes/sections/browser-recommendation.html

```diff
@@ -23,10 +23,99 @@ <h1 id="browser" class="anchor"><a href="#browser"><i class="fas fa-link anchor-
 website="https://www.torproject.org/"
 forum="https://forum.privacytools.io/t/discussion-tor-browser/278"
 git="https://trac.torproject.org/projects/tor"
-windows=""
-mac=""
-linux=""
-android=""
-ios=""
-bsd=""
+windows="https://www.torproject.org/download/"
+mac="https://www.torproject.org/download/"
+linux="https://www.torproject.org/download/"
+android="https://play.google.com/store/apps/details?id=org.torproject.torbrowser&hl=en_US"
 %}
+
+<h1 id="browser-android" class="anchor"><a href="#browser-android"><i class="fas fa-link anchor-icon"></i></a> Browser Recommendations For Android</h1>
+
+{% include cardv2.html
+title="Firefox Browser"
+image="/assets/img/tools/Firefox.png"
+description='Firefox is fast, reliable, open source and respects your privacy. Don\'t forget to adjust the settings according to our
+recommendations: <a href="#webrtc"><i class="fas fa-link"></i> WebRTC</a> and <a href="#about_config"><i class="fas fa-link"></i> about:config</a> and get the <a href="#addons"><i class="fas fa-link"></i> privacy add-ons</a>.'
+website="https://www.mozilla.org/en-US/firefox/mobile/"
+forum="https://forum.privacytools.io/t/discussion-firefox/279"
+source="https://github.com/mozilla-mobile"
+windows="https://www.mozilla.org/en-US/firefox/windows/"
+mac="https://www.mozilla.org/en-US/firefox/mac/"
+linux="https://www.mozilla.org/en-US/firefox/linux/"
+android="https://play.google.com/store/apps/details?id=org.mozilla.firefox&hl=en_US"
+ios="https://apps.apple.com/us/app/firefox-private-safe-browser/id989804926"
+bsd="https://www.freebsd.org/doc/handbook/desktop-browsers.html"
+%}
+
+{% include cardv2.html
+title="Tor Browser For Android - Provides Anonymity"
```

Comment:
```suggestion
title="Tor Browser - Provides Anonymity"
```

Seems long, we're in the Android section already

Reply:
Too my knowledge, the official name is Tor Browser for android.

Reply:
<img width="779" alt="image" src="https://user-images.githubusercontent.com/3637842/64295942-47b9fc00-cf38-11e9-9b42-4787ab11079b.png">

Regardless I'm less concerned about official names and more concerned about "clear" names.

Reply:
Hmm, odd, the alpha application says: Tor browser for android (alpha). Maybe it was changed?
In any case, you seem to be correct here, ill change it.